ontology/yaml/resources/HVAC/entity_types/CDWS.yaml

```diff
@@ -16,6 +16,13 @@
 ### Canonical Types  ###
 ########################
 
+CDWS_CWDT:
```

Comment:
Please confirm there is no temperature control on this loop

Reply:
@cstirdivant - thank you, changed to WDT there. Will confirm re temperature control ASAP.

Reply:
@Ciaran-Oflannagain any update on if there is supply (or return) water temperature control on this system? How does it know what temperature of water to provide to the chillers?

Reply:
@cstirdivant - Apologies still awaiting clarity from BMS Contractor.

Reply:
@cstirdivant - I can confirm there is no temperature control on this packaged system

Reply:
@cstirdivant - is there any further information needed for this CDWS type? Thanks, Ciaran

Reply:
@cstirdivant - just following up on this PR. Thanks